t/39-tinycv.t

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/perl
+# Checks units provided by tinycv.pm module.
+# TODO: This test does not cover all provided functions yet.
```

Comment:
I'd say it's ok to not have all included. Please don't include "TODOs" in the code. Those tend to stay forever :) If you want to remember ideas where to improve have a ticket.